docs/source/api/python/sampling.rst

```diff
@@ -30,3 +30,15 @@ Builtin sampler classes for more complicated sampling algorithms
 ----------------------------------------------------------------
 .. autoclass:: RandomWalkNeighborSampler
 .. autoclass:: PinSAGESampler
+
+Neighborhood samplers for multilayer GNNs
+-----------------------------------------
+.. autoclass:: MultiLayerNeighborSampler
+
+Data loaders for minibatch iteration
+------------------------------------
+.. autoclass:: NodeDataLoader
```

Comment:
We have NodeCollator instead of NodeDataLoader, right?

Reply:
Right.  Let me fix it...